x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/openai/OpenAiResponseHandler.java

```diff
@@ -35,6 +35,8 @@ public class OpenAiResponseHandler extends BaseResponseHandler {
     static final String REMAINING_TOKENS = "x-ratelimit-remaining-tokens";
 
     static final String CONTENT_TOO_LARGE_MESSAGE = "Please reduce your prompt; or completion length.";
+
+    static final String INPUT_OR_OUTPUT_TOKENS_MUST_BE_REDUCED = "The input or output tokens must be reduced in order to run successfully";
```

Comment:
I think we can remove this now right?

Reply:
Good spot, interesting that `spotlessApply` doesn't do this for me though